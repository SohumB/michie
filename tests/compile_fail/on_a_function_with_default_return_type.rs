use michie::memoized;

#[memoized(key_expr = ())]
fn f() {}

fn main() {}
