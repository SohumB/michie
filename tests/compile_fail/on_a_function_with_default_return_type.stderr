error: custom attribute panicked
 --> tests/compile_fail/on_a_function_with_default_return_type.rs:3:1
  |
3 | #[caching(key_expr = ())]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: not implemented: default return types are not supported
